docs/docs-beta/docs/guides/data-modeling/metadata.md

```diff
@@ -30,30 +30,34 @@ To follow the steps in this guide, you'll need:
 
 ## Adding owners to your assets
 
-In a large organization, it's important to know who is responsible for a given data asset. With `owners` it's straightforward to add individuals and teams as owners for your asset:
+In a large organization, it's important to know which individuals and teams are responsible for a given data asset:
 
 <CodeExample filePath="guides/data-modeling/metadata/owners.py" language="python" title="Using owners" />
 
 `owners` must either be an email address, or a team name prefixed by `team:`.
 
-> With Dagster+ Pro, you can create asset-based alerts that will automatically notify an asset's owners when triggered. Refer to the [Dagster+ alert documentation](/dagster-plus/deployment/alerts) for more information.
+:::tip
 
+With Dagster+ Pro, you can create asset-based alerts that will automatically notify an asset's owners when triggered. Refer to the [Dagster+ alert documentation](/dagster-plus/deployment/alerts) for more information.
 
+:::
 
 ## Choosing between tags or metadata for custom information
 
 In Dagster, you can attach custom information to assets in two ways: **tags** and **metadata**.
 
-**Tags** are a simple way to organize assets in Dagster. You can attach several tags to an asset when it is defined, and they will appear in the UI. You can also use tags to search and filter for assets in the [asset catalog](/todo). They are structured as key-value pairs of strings.
+**Tags** are the primary way to organize assets in Dagster. You can attach several tags to an asset when it's defined, and they will appear in the UI. You can also use tags to search and filter for assets in the [Asset catalog](/todo). They're structured as key-value pairs of strings.
 
 Here's an example of some tags one might apply to an asset:
```

Comment:
Nit - can we switch `one` to `you`, even though it's technically not part of this PR? It's giving me college essay vibes